UPGRADE-4.4.md

```diff
@@ -86,6 +86,7 @@ FrameworkBundle
  * The `ControllerResolver` and `DelegatingLoader` classes have been marked as `final`.
  * The `controller_name_converter` and `resolve_controller_name_subscriber` services have been deprecated.
  * Deprecated `routing.loader.service`, use `routing.loader.container` instead.
+ * Not tagging service route loaders with `routing.route_loader` has been deprecated.
```

Comment:
please upgrade UPGRADE-5.0.md too

Reply:
I planned to upgrade it in the PR that will remove the deprecated code.

Reply:
that would be on master, while the notice must be on 4.4